apps/core/scenes/bookings/BookingBadges.js

```diff
@@ -1,5 +1,6 @@
 // @flow
 
+/* eslint-disable react-native/no-unused-styles */ // Style is not unused, eslint just don't pick it up when used this way
```

Comment:
One way to do that otherwise is to be a bit more verbose and add a switch statement to make sure all styles are being used. Not sure what's the best approach here.

Reply:
At least with the switch, there are no surprises.

Reply:
If the switch shut's up the eslint rule, that is definitely better. I will give it a shot